js/app/restaurant/components/Order/MobileOrderHeading.js

```diff
@@ -39,7 +42,7 @@ export default function MobileOrderHeading() {
         }) }
         onClick={ handleClick }>
         <div className="panel-heading__body">
-          { isMobileCartVisible ? null : (<OrderState />) }
+          { isMobileCartVisible ? (<span>{t('CART_TITLE')}</span>) : (<OrderState />) }
```

Comment:
Mobile cart title is yellow like a warning it is expected ?

![image](https://github.com/coopcycle/coopcycle-web/assets/4426585/2e0b951a-9222-42c5-a408-d09ca1ae23ff)

Reply:
Yes, it's the same `<OrderState />` component 😄